lib/vm/src/vmcontext.rs

```diff
@@ -15,15 +15,37 @@ use std::ptr::{self, NonNull};
 use std::sync::Arc;
 use std::u32;
 
+/// We stop lying about what this daat is
+/// TODO:
```

Comment:
Did you mean to include these? A comment about "the first parameter may be a VMContext parameter for a wasm function call, or an arbitrary pointer produced by the host for host function calls" might make sense here?

Reply:
Those were dev-comments, that I blindly ported over from the other PR, yeah will clean up